include/Framework/EventHeader.h

```diff
@@ -131,6 +190,12 @@ class EventHeader {
 
   /**
    * Set the event weight.
+   * 
+   * The event weight is by default 1. for all events. It is up to
```

Comment:
```suggestion
   * The event weight is by default 1 for all events. It is up to
```
remove `.` for reading clarity